hail/python/hail/ir/matrix_ir.py

```diff
@@ -167,18 +167,27 @@ def __init__(self, child, pred):
     def render(self, r):
         return '(MatrixFilterEntries {} {})'.format(r(self.child), r(self.pred))
 
+class MatrixKeyRowsBy(MatrixIR):
+    def __init__(self, child, keys, is_sorted=False):
+        super().__init__()
+        self.child = child
+        self.keys = keys
+        self.is_sorted = is_sorted
+
+    def render(self, r):
+        return '(MatrixKeyRowsBy ({}) {} {})'.format(
+            ' '.join([escape_id(x) for x in self.keys]),
```

Comment:
Why are you using `escape_id` instead of `escape_str` as before?

Reply:
escape_id is correct here, since that's what's done in Pretty/Parser